sdks/python/apache_beam/runners/common.py

```diff
@@ -22,8 +22,13 @@
 For internal use only; no backwards-compatibility guarantees.
```

Comment:
Since this file will be cythonized, let's tell cython to use Python3 semantics and add:

` # cython: language_level=3,`

See also: https://github.com/apache/beam/blob/7c3fba01c9a69aca2a75bec52572d097a6f9c777/sdks/python/setup.py#L168

Reply:
This has some interesting consequences for typechecks on strings.
```python
if not isinstance(tag, six.string_types):
``` 
cannot be replaced by 
```python
if not isinstance(tag, (str, unicode)):
```
Since the string objects defined in other modules (in Python2) are seen as bytes in Cython code with Python3 semantics.

Therefore I replaced it by 
```python
try:
    basestring
except NameError:
    basestring = str
```
and 
```python
if not isinstance(tag,basestring):
```